### PR TITLE
AB#86129 Added users list and join requests list

### PR DESCRIPTION
**Pull request by Antek012**
image

### CODE REVIEW

File: src/HE.Investments.Account.Domain/Users/QueryHandlers/GetUsersAndJoinRequestsQueryHandler.cs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {
+        var organisation = await _organizationRepository.GetBasicInformation(await _accountUserContext.GetSelectedAccount(), cancellationToken);
+        var users = await _usersRepository.GetUsers();
+
+        return new UsersAndJoinRequests(
+            organisation.RegisteredCompanyName,
+            users,
+            new List<UserDetails>
+            {
+                new("11", "Tomasz", "Kot", "kot@elo.pl", "nikt", "Limited", null),
```
Comment (Antek012) [inline src/HE.Investments.Account.Domain/Users/QueryHandlers/GetUsersAndJoinRequestsQueryHandler.cs:33]: Test data for now

File: src/HE.Investments.Account.Domain/Users/Repositories/UsersRepository.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    private readonly IUsersCrmContext _usersCrmContext;
+
+    public UsersRepository(IUsersCrmContext usersCrmContext)
+    {
+        _usersCrmContext = usersCrmContext;
+    }
+
+    public async Task<IList<UserDetails>> GetUsers()
+    {
+        var users = await _usersCrmContext.GetUsers(PortalConstants.LoansPortalType);
```
Comment (Antek012) [inline src/HE.Investments.Account.Domain/Users/Repositories/UsersRepository.cs:18]: Change to AHP

File: src/HE.Investments.Account.WWW/Controllers/UsersController.cs [hunk 3]
```diff
@@ -0,0 +1,32 @@
+using HE.Investments.Account.Contract.Users.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace HE.Investments.Account.WWW.Controllers;
+
+[Authorize]
```
Comment (Carq) [inline src/HE.Investments.Account.WWW/Controllers/UsersController.cs:8]: Authorize with completed profile

Comment (Antek012) [reply src/HE.Investments.Account.WWW/Controllers/UsersController.cs]: changed

**Pull request merged**